adapters/data_parser.py

```diff
@@ -1,26 +1,27 @@
 import datetime
 
-from ports.custom_errors import CustomErrors
+from ports.data_parser import UnsupportedFileFormatError, FileNotFound, AccessDeniedError, BinaryFileError, \
+    EmptyFileError, InvalidGradeError, DataProcessingError, UnknownError
 from ports.student import Student
 
 
 def read_scores(file_path: str) -> list[Student]:
     try:
         if not file_path.lower().endswith('.txt'):
-            raise CustomErrors.UnsupportedFileFormatError()
+            raise UnsupportedFileFormatError()
```

Comment:
parentheses not needed when raising an exception without providing any arguments. Here and in other places